src/examples/mfem_common_plane.cpp

```diff
@@ -0,0 +1,429 @@
+    mfem::Vector f_true {fespace.GetTrueVSize()};
+    P.MultTranspose(f, f_true);
+    P.Mult(f_true, f);
+
+    // force already summed over ranks
+    f.Add(1.0, tribol::getMfemResponse(0));
+
+    for (int i {0}; i < inv_lumped_mass.Size(); ++i)
+    {
+      a[i] = inv_lumped_mass[i] * f[i];
+    }
+  }
+
+private:
+  mfem::ParBilinearForm elasticity;
+  std::unique_ptr<mfem::ParBilinearForm> damping;
+  mfem::Vector inv_lumped_mass;
+};
+
+int main( int argc, char** argv )
+{
+  // initialize MPI
+  MPI_Init( &argc, &argv );
+  int np, rank;
+  MPI_Comm_size(MPI_COMM_WORLD, &np);
+  MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+
+#ifdef TRIBOL_USE_UMPIRE
+  umpire::ResourceManager::getInstance();  // initialize umpire's ResouceManager
+#endif
+
+  // initialize logger
+  axom::slic::SimpleLogger logger;
+  axom::slic::setIsRoot(rank == 0);
+
+  // command line options
+  // number of times to uniformly refine the serial mesh before constructing the
+  // parallel mesh
+  int ref_levels = 2;
+  // polynomial order of the finite element discretization
+  int order = 1;
+  // initial velocity
+  double initial_v = 0.001;
+  // timestep size
+  double dt = 0.001;
+  // end time
+  double t_end = 0.35;
+  // kinematic penalty
+  double p_kine = 500.0;
+  // number of cycles to skip before output
+  int output_cycles = 5;
+
+  axom::CLI::App app { "mfem_common_plane" };
+  app.add_option("-r,--refine", ref_levels,
+    "Number of times to refine the mesh uniformly.")
+    ->capture_default_str();
+  app.add_option("-o,--order", order, 
+    "Finite element order (polynomial degree).")
+    ->capture_default_str();
+  app.add_option("-v,--initialv", initial_v, 
+    "Initial velocity of the top block.")
+    ->capture_default_str();
+  app.add_option("-d,--dt", dt, 
+    "Timestep size.")
+    ->capture_default_str();
+  app.add_option("-e,--endtime", t_end, 
+    "Time of the end of the simulation.")
+    ->capture_default_str();
+  app.add_option("-p,--kinematicpenalty", p_kine, 
+    "Kinematic penalty parameter.")
+    ->capture_default_str();
+  app.add_option("-c,--outputcycles", output_cycles, 
+    "Cycles to skip before next output.")
+    ->capture_default_str();
+  CLI11_PARSE(app, argc, argv);
+
+  SLIC_INFO_ROOT("Running mfem_common_plane with the following options:");
+  SLIC_INFO_ROOT(axom::fmt::format("refine: {0}", ref_levels));
+  SLIC_INFO_ROOT(axom::fmt::format("order:  {0}\n", order));
+
+  // fixed options
+  // location of mesh file. TRIBOL_REPO_DIR is defined in tribol/config.hpp
+  std::string mesh_file = TRIBOL_REPO_DIR "/data/two_hex_apart.mesh";
+  // boundary element attributes of surface 1
+  auto surf1_attribs = std::set<int>({4});
+  // boundary element attributes of surface 2
+  auto surf2_attribs = std::set<int>({5});
+  // boundary element attributes of fixed surface (all t)
+  auto fix_attribs = std::set<int>({3});
+  // element attributes of moving volumes (initial condition)
```

Comment:
Elaborate. Is this the element attributes of all elements with a specified initial velocity?

Reply:
Correct; comment has been updated.